src/wrappers/start.js

```diff
@@ -11,10 +11,10 @@
                     jQuery = require('jquery')(root);
```

Comment:
Maybe we should define other dependencies similarly like jQuery here, in case there are not already required